src/shared/components/app/navbar.tsx

```diff
@@ -448,8 +450,13 @@ export class Navbar extends Component<NavbarProps, NavbarState> {
                   <li className="nav-item">
                     <NavLink
                       to="/signup"
-                      className="nav-link"
-                      title={I18NextService.i18n.t("sign_up")}
+                      className={
+                        registrationClosed ? "nav-link pe-none" : "nav-link"
+                      }
+                      aria-disabled={registrationClosed}
+                      title={I18NextService.i18n.t(
+                        registrationClosed ? "registration_closed" : "sign_up",
+                      )}
```

Comment:
It seems like both `pe-none` and `aria-disabled` prevent title from being shown.

Reply:
I second Dessalines on this. Normally, disabling an input communicates that an input is temporarily unavailable (like when a form is submitting), or it is used to prevent a user from putting the form in an incorrect state (like how the send notifications to email checkbox is disabled when no email is specified). Hiding an input is more appropriate when a user shouldn't have access to a functionality in the first place, like how mod and admin actions aren't shown for non mod/admin users. I think a signup button on an instance with registration disabled falls under the latter.

Reply:
Okay done